File: sync/rest/sync_maps/update-map-item/update-map-item.rb
require 'http'
require 'twilio-ruby'

# Initialize the client
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
client = Twilio::REST::Client.new(account_sid, auth_token)

# Retreive the service
service = client.preview.sync.services('SERVICE_SID')

# Update a Map Item
response = service.sync_maps('Players').sync_map_items.update(key:'steph_curry', data: "{ 'name': 'Stephen Curry', 'level': '31', 'username':'spicy_curry'}")
puts response

File: sync/rest/maps/delete-map-item/update-map-item.cs
// Download the next-gen twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio.Clients;
using Twilio.Resources.Preview.Sync.Service.SyncMap;

class Example
{
  static void Main(string[] args)
  {
    // Find your Account Sid and Auth Token at twilio.com/console
    var accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    var authToken = "your_auth_token";
    var client = new TwilioRestClient(accountSid, authToken);

    SyncMapItemResource.Delete("ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
      "Players", "steph_curry")
      .Execute(client);

    Console.WriteLine("Deleted Item");
  }
}